crates/ruff/src/rules/ruff/rules/confusables.rs

```diff
@@ -2116,5 +2116,8 @@ pub(crate) static CONFUSABLES: Lazy<FxHashMap<u32, u8>> = Lazy::new(|| {
         (1059, 89),
         (65283, 35),
         (65307, 59),
+        (0x212B, 0x00C5), // ANGSTROM SIGN → LATIN CAPITAL LETTER A WITH RING ABOVE
+        (0x2126, 0x03A9), // OHM SIGN → GREEK CAPITAL LETTER OMEGA
+        (0x00B5, 0x03BC), // MICRO SIGN → GREEK SMALL LETTER MU
```

Comment:
Based on Section 2.5 in https://www.unicode.org/reports/tr25/, should this also include Kelvin sign?

Reply:
The list from VS Code covers Kelvin with `(8490, 75)`.